jupyterlab_bigquery/src/components/details_panel/table_details_tabs.tsx

```diff
@@ -86,7 +86,10 @@ export default class TableDetailsTabs extends React.Component<Props, State> {
                     'main',
                     queryId,
                     undefined,
-                    [queryId, `SELECT * FROM \`${this.props.table_id}\``]
+                    [
+                      queryId,
+                      `SELECT * FROM \`${this.props.table_id}\` LIMIT 1000`,
+                    ]
```

Comment:
I changed this to `LIMIT 1000` to mimic the web UI behavior

Reply:
Thanks this was a friction point that makes this more usable